source/support.tex

```diff
@@ -2877,19 +2877,19 @@
 in storage occupied by an existing object of the same type,
 a pointer to the original object
 can be used to refer to the new object
-unless the type contains \tcode{const} or reference members;
-in the latter cases,
+unless its complete object is a const object;
```

Comment:
Another remaining case that we might want to call out here is if the object is a base class subobject.

Reply:
Fixed.